src/components/CandidateForm.jsx

```diff
@@ -23,8 +30,12 @@ const CandidateForm = ({ fields, onSubmit }) => {
 
   const onSelectChange = (selected) => {
     // 3. copy values
+    const newValues = [...values];
     // find index of technologies input
+    const index = newValues.findIndex((value) => value.name === "technologies");
     // loop through the selected values
+    newValues[index] = { name: "technologies", value: selected };
```

Comment:
In here you should .map over the `selected` values, because we only need the ids of the technologies. 

`value: selected.map((s) => s.value),`